app/models/image.rb

```diff
@@ -10,7 +10,7 @@ class Image < ActiveFedora::Base
   self.indexer = ImageIndexer
 
   validates :title, presence: { message: 'Your work must have a title.' }
-  validates :accession_number, presence: { message: 'Your work must have an accession number.' }
+  validates :accession_number, presence: { message: 'Accession number is required.' }, accession_number: true, on: :create
```

Comment:
ok this line is what i meant, i think we can get rid of the `on: :create` since we want to always make sure it's unique

Reply:
I don't think so. During the create process it seems to hit the validation and/or save multiple times and I was getting an error anytime I tried to create a work.

Reply:
haha oh well, then yeah it's good